SRC/PORT_IMAGE.CPP

```diff
@@ -3,6 +3,13 @@
 #include <SDL_surface.h>
 #include <SDL_image.h>
 #include <ERROR/ERROR.H>
+#include <climits>
+
+#ifdef _MSC_VER
+#ifndef PATH_MAX
+#define PATH_MAX 260
```

Comment:
Windows has `MAX_PATH`. Trouble is that `windows.h` cannot be easily included from anywhere in TK codebase due to type clashes. It could be done with function in `MISCFUNC_MSVC` which has needed include.

But maybe it's overkill because the limit is unlikely ever going down. :)